src/base/rss/rss_autodownloadrule.cpp

```diff
@@ -64,7 +64,7 @@ namespace
 
     QJsonValue triStateBoolToJsonValue(const TriStateBool &triStateBool)
     {
-        switch (static_cast<int>(triStateBool)) {
+        switch (triStateBool.value()) {
```

Comment:
It's incorrect accessor for classes like this one. It doesn't just hold value of some type (like, e.g., boost::optional). So I strongly prefer to keep existing behavior (allow type casting to integer). Of course, you can change it to `operator signed char()` if you think it's going to do any good.

Reply:
OK changed.